content/en/docs/guides/server-graceful-stop.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Graceful Shutdown
+description: >-
+  Explains how to gracefully shut down [or stop, if you prefer] a gRPC server
```

Comment:
The brackets were there for you to read and choose either "stop" or "shut down", not to copy verbatim.  :)

Reply:
Ah yeah. Forgot to remove. Removed now.